validator/src/consensus/notifier_ffi.rs

```diff
@@ -15,396 +15,395 @@
  * ------------------------------------------------------------------------------
  */
 
- use std::ffi::CStr;
```

Comment:
In this commit "Fix unused return value warnings", its impossible to verify the changes because of the format changes. Were the formatting changes fixed here introduced in a previous commit?

Reply:
Do the changes I've made resolve the issue for you?